### PR TITLE
Deploy binary packages to GitHub Releases

### PR DESCRIPTION
**Pull request by dtan4**
## WHY

To download stable binaries from anywhere.

## WHAT

Deploy binary packages (tarball, zipball) to GitHub Releases with git tag.

All binaries are statically linked.

```bash
$ find dist -type f -name 'slack-notifier' -exec file {} \;
dist/darwin-386/slack-notifier: Mach-O executable i386
dist/darwin-amd64/slack-notifier: Mach-O 64-bit executable x86_64
dist/linux-386/slack-notifier: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), statically linked, stripped
dist/linux-amd64/slack-notifier: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, stripped
dist/windows-386/slack-notifier: PE32 executable for MS Windows (console) Intel 80386 32-bit
dist/windows-amd64/slack-notifier: PE32+ executable for MS Windows (console) Mono/.Net assembly
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,13 +11,29 @@ env:
...
+  - sudo apt-get update -q
+  - sudo apt-get install glide -y
+  - mkdir -p $GOPATH/bin
+install:
+  - make deps
 script:
   - echo ${TRAVIS_COMMIT::8}
   - make ci-build
   - make test
 deploy:
```
Comment (koudaiii) [inline .travis.yml]: I think need cross-build. 👀 
Do you think?

```yaml
before_deploy:
  - make cross-build
  - make dist
```

Comment (dtan4) [reply .travis.yml]: You're right, I'll add it.

Comment (dtan4) [reply .travis.yml]: Added at https://github.com/wantedly/slack-notifier/pull/8/commits/921327e113d97ffc65fab65301fbe161148e19ad

**Comment by koudaiii** [comment]
LGTM!!

**Comment by dtan4** [comment]
Deployed successfully.
https://github.com/wantedly/slack-notifier/releases/tag/v0.1.0

**Pull request merged**